ocp-templates/ibutsu-server-openshift.yaml

```diff
@@ -66,7 +26,12 @@ objects:
           deploymentConfig: ibutsu-frontend
       spec:
         containers:
-        - image: ibutsu-frontend
+        - env:
+            - name: REACT_APP_SERVER_URL
+              value: ${BACKEND_ROUTE}
+            - name: NODE_ENV
+              value: production
+        - image: frontend
```

Comment:
I'm not 100% sure if this is necessary or if it needs to be `quay.io/ibutsu/frontend`?

Reply:
Hrm, yes, I think it needs to be `quay.io/ibutsu/frontend` -- try it and see if there are any problems :-)

Reply:
Looking into this a bit more, it seems like for native OCP objects (DeploymentConfig's included) it is enough to just use the image stream name as we are here. OCP will resolve the image from the image stream name.

For Deployments, if we wanted to enable this we would have to change the `spec.lookupPolicy.local` to `true`. 

https://docs.openshift.com/container-platform/4.6/openshift_images/using-imagestreams-with-kube-resources.html
https://austindewey.com/2020/10/25/you-can-use-openshifts-imagestreams-with-deployments-heres-how/